department-of-veterans-affairs/abd-vro
# added ignore property

## What was the problem?
unrecognized field when jackson json does the serialization/deserialization 

Associated tickets or Slack threads:
- #2749 

## How does this fix it?[^1]
Added the annotation to ignore the unknown properties so we don't run into exception in apache camel

## How to test this PR
- deploy and check `xample-workflow` log to see if we experience the same exception again


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
